app/helpers/mailer_helper.rb

```diff
@@ -186,7 +186,7 @@ def commenter_pseud_or_name_link(comment)
     elsif comment.by_anonymous_creator?
       style_bold(t("roles.anonymous_creator"))
     else
-      style_link(comment.comment_owner_name, polymorphic_url(comment.comment_owner, only_path: false))
+      style_bold(style_link(comment.comment_owner_name, polymorphic_url(comment.comment_owner, only_path: false)))
```

Comment:
@sarken pointed out to me that we have a helper that can help here: https://github.com/otwcode/otwarchive/blob/93184cec8d7954bc6085153f375ae91de57b7e9e/app/helpers/mailer_helper.rb#L37-L44 (it does include the kudos icon, but we're OK with that as another way to distinguish registered users from guests). It seems to play nicely with `style_bold` but we're fine as long as one of the icon or bold is there